oppijanumerorekisteri-domain/src/main/java/fi/vm/sade/oppijanumerorekisteri/repositories/HenkiloRepository.java

```diff
@@ -20,6 +20,6 @@
 
     Optional<Henkilo> findByOidHenkilo(String henkiloOid);
 
-    Optional<Henkilo> findByHetu(String hetu);
+    Optional<Henkilo> findFirstByHetu(String hetu);
```

Comment:
Sellaisia tapauksia (poislukien testihetut ja poistetut) on kyllä muutamia, kellä on useita hetuja. Tällöin käytös ei tosiaan ole ennustettavaa mistä voi syntyä omituisuuksia. Ainakin nykyisin henkilöpalvelussa tehdään samoin eli palautetaan vain ensimmäinen. 

Vaikka tästä saattaakin koitua opiskelijoiden puolelle jotakin häikkää niin 500 palauttaminen on varmasti silti parempi kuin että hakijat menevät sekaisin. Täytyy heittää hallittu virhe.